src/testUtils.tsx

```diff
@@ -130,6 +130,15 @@ export function renderWithProvider(
     return { client: mockedClient, realtime: mockedRealtime, rerender, rendered, reset };
 }
 
+export async function actAndFlush<T>(fn: () => T): Promise<T> {
```

Comment:
could we include a comment explaining this

Reply:
Yea I meant to and completely forgot to come back and add, it took me forever to get this stuff straight.